README.md

```diff
@@ -1,38 +1,50 @@
-jupyter-vim-binding
-========================
-Enable Vim key bindings in [jupyter](https://jupyter.org/) or [IPython notebook](http://ipython.org/notebook.html).
+#Jupyter VIM Binding
+
+This extension enables VIM keybindings in [Jupyter Notebook](https://jupyter.org/) (4.0+).
```

Comment:
According to IPython (http://ipython.org/news.html)

> The 3.x release series will be the last where IPython is released as one big package. For 4.0, we will split up components into several packages. The parts which work for any language will be called Jupyter, while the parts specific to executing Python code will remain as Python.

So, starting 4.0, isn't it only Jupyter Notebook?